softwerkskammer/test/groups/group_object_test.js

```diff
@@ -192,3 +192,17 @@ describe('returns the meetup :urlname from the given meetup URL', () => {
   });
 
 });
+
+describe('guests can contact the organizers', () => {
+  it('for spam projection purposes is an opt in feature', () => {
```

Comment:
When i first read the test I was a bit confused what the two different examples meant (now I know :-) ). Should we break this up in two different tests?

Reply:
done